server/main.lua

```diff
@@ -70,32 +67,33 @@ end
 
 exports('UpdateVehicleEntity', updateEntity)
 
+---@class SetEntityOwnerQuery
+---@field citizenId string
+---@field vehiclePlate string
+
 --- Update Vehicle Entity Owner
----@param citizenId string
----@param license string
----@param vehiclePlate string
-local function updateEntityOwner(citizenId, license, vehiclePlate)
-    MySQL.update('UPDATE player_vehicles SET citizenid = ?, license = ? WHERE plate = ?', {
```

Comment:
License is still a required field. So still needs to be set, just do a lookup or join with the players table to get the license of the citizenid.